### ISSUE TITLE
Configure which Pokemon to catch and which to ignore.

### ISSUE THREAD

**Issue by Majamulli** [AnthonyOSX/GO-Bot#219]
Is there a way to do this? If not I'd suggest it. I currently have ~2500 zubat candys and seriously dont want any more. Would like to use this bot now to get "rarer" pokemon if possible.

**Comment by cawk** [comment]
if the source for 0.05 gets shared this is a feature i am willing to add myself obviously i will share a commit for it but just awaiting the 0.05 source

**Comment by Connor-Davies** [comment]
You're aware you still get stardust for each pokemon you catch right?  Why would you want to catch fewer of them?

**Comment by Majamulli** [comment]
Because the amount of pokemon that get caught in a certain time is limited. In big citys there are enough pokemon to keep the bot busy without catching the useless stuff. So I'd rather the bot picked the rarer pokemon and it would still have the same stardust and exp effect.

**Comment by sweynforkbeard** [comment]
Theyre not useless -_-

**Comment by Majamulli** [comment]
Okay I am sorry for being politically incorrect. I meant to say that they are indeed very useful but I'd rather have the rarer ones because for me personally they are even more useful! Happy now?

**Comment by Bokbunja** [comment]
Evolve all your zubats for a massive experience gain with a lucky egg

**Comment by Majamulli** [comment]
But I'd rather evolve all my Charmanders and Squirtles instead. Much more fun. Come on, what's with this discussion? I obviously know that you can use the candies for evolving and that it's useful to catch everything because of stardust and exp. But that's not what I am after. I used this bot for some time and am currently level 27 and would like to concentrate more on getting the rarer pokemon. I am not interested in getting to level 40 as soon as possible, I am more interested in getting good pokemon and for that it would be better to catch the rarer pokemon instead of more and more of the pidgey and zubats.

**Comment by rayn21** [comment]
I agree with @cawk idea about enhancing v5. But I'm getting inpatient since we're no longer open source and I'm considering forking the necro bot and moving forward on that.
